seacrowd/sea_datasets/yunshan_cup_2020/yunshan_cup_2020.py

```diff
@@ -0,0 +1,180 @@
+
+_DESCRIPTION = """\
+Lao POS dataset containing 11,000 sentences was released as part of Yunshan-Cup-2020 evaluation track.
+"""
+
+_HOMEPAGE = "https://github.com/GKLMIP/Yunshan-Cup-2020"
+
+_LICENSE = Licenses.UNKNOWN.value  # example: Licenses.MIT.value, Licenses.CC_BY_NC_SA_4_0.value, Licenses.UNLICENSE.value, Licenses.UNKNOWN.value
+
+_URLS = {
+    "train": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/train.txt",
+    "dev": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/dev.txt",
+    "test": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/test.txt",
+}
+_SUPPORTED_TASKS = [Tasks.POS_TAGGING]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class YunshanCup2020(datasets.GeneratorBasedBuilder):
+    """Lao POS dataset containing 11,000 sentences was released as part of Yunshan-Cup-2020 evaluation track."""
+
+    class_labels = ["IAC", "COJ", "ONM", "PRE", "PRS", "V", "DBQ", "IBQ", "FIX", "N", "ADJ", "DMN", "IAQ", "CLF", "PRA", "DAN", "NEG", "NTR", "REL", "PVA", "TTL", "DAQ", "PRN", "ADV", "PUNCT", "CNM"]
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="yunshan_cup_2020_source",
+            version=SOURCE_VERSION,
+            description="yunshan_cup_2020 source schema",
+            schema="source",
+            subset_id="yunshan_cup_2020",
+        ),
+        SEACrowdConfig(
+            name="yunshan_cup_2020_seacrowd_seq_label",
+            version=SEACROWD_VERSION,
+            description="yunshan_cup_2020 SEACrowd schema",
+            schema="seacrowd_seq_label",
+            subset_id="yunshan_cup_2020",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "yunshan_cup_2020_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "index": datasets.Value("string"),
+                    "tokens": [datasets.Value("string")],
+                    "pos_tags": [datasets.Value("string")],
+                }
+            )
+        elif self.config.schema == "seacrowd_seq_label":
+            features = schemas.seq_label_features(self.class_labels)
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        train_path = dl_manager.download_and_extract(_URLS["train"])
+        dev_path = dl_manager.download_and_extract(_URLS["dev"])
+        test_path = dl_manager.download_and_extract(_URLS["test"])
```

Comment:
can use the following:
```suggestion
path_dict = dl_manager.download_and_extract(_URLS)
        train_path, val_path, test_path = path_dict["train"], path_dict["val"], path_dict["test"]
```